### PR TITLE
Add 'breakpad' package

### PR DESCRIPTION
**Pull request by t0p4**
* I've followed [this guide](https://docs.hunter.sh/en/latest/creating-new/create/cmake.html)
  step by step carefully. **[Yes]**

* I've tested this package remotely and have excluded all broken builds.
  Here is the links to the Travis/AppVeyor with status "All passed":

  * https://ci.appveyor.com/project/t0p4/hunter-7qcko/builds/20553791
  * https://travis-ci.org/t0p4/hunter/builds/459721406

* I have submitted CI configs to https://github.com/ingenue/hunter targeting `pkg.template` branch,
  see this merged pull request https://github.com/ingenue/hunter/pull/297

---

### CODE REVIEW

File: cmake/projects/breakpad/hunter.cmake [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    breakpad
+    VERSION
+    1.0-p0
+    URL
+    "https://github.com/t0p4/breakpad/archive/v1.0-p0.tar.gz"
```
Comment (ruslo) [inline cmake/projects/breakpad/hunter.cmake]: Please send pull request with changes to the fork: https://github.com/hunter-packages/breakpad

It make sense to deal with it before submitting main pull request.

Comment (t0p4) [reply cmake/projects/breakpad/hunter.cmake]: I didn't find information about it. what do you advise

Comment (ruslo) [reply cmake/projects/breakpad/hunter.cmake]: > I didn't find information about it. what do you advise

You mean there was no breakpad fork before? You can open "New package" issue and mention there that we need fork.

Comment (t0p4) [reply cmake/projects/breakpad/hunter.cmake]: I mean about package version

File: cmake/configs/default.cmake [hunk 2]
```diff
@@ -167,6 +167,7 @@ hunter_default_version(autoutils VERSION 0.3.0)
 hunter_default_version(benchmark VERSION 1.4.0)
 hunter_default_version(bison VERSION 3.0.4-p0)
 hunter_default_version(boost-pba VERSION 1.0.0-p0)
+hunter_default_version(breakpad VERSION 1.0-p0)
```
Comment (ruslo) [inline cmake/configs/default.cmake]: Wrong version

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/builds/20578571
* https://travis-ci.org/ingenue/hunter/builds/460139585?utm_source=github_status&utm_medium=notification

**Comment by t0p4** [comment]
I don't specify VERSION

**Comment by ruslo** [comment]
Related: https://github.com/hunter-packages/breakpad/pull/3

**Comment by t0p4** [comment]
Related: https://github.com/hunter-packages/breakpad/pull/4

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/builds/20644382
* https://travis-ci.org/ingenue/hunter/builds/461242888?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Uploading:
* https://ci.appveyor.com/project/ingenue/hunter/builds/20645398
* https://travis-ci.org/ingenue/hunter/builds/461259902?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.23.60

**Pull request merged**